TramsDataApi/Migrations/TramsDb/20221104152301_Remove Concerns Casework tables.cs

```diff
@@ -0,0 +1,1426 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace TramsDataApi.Migrations.TramsDb
+{
+    public partial class RemoveConcernsCaseworktables : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
```

Comment:
I take you have moved the tables into a different db schema? And migrated the data>